routes/subscribe/index.js

```diff
@@ -0,0 +1,52 @@
+const subscribe = require('express').Router();
+const request = require('request');
+
```

Comment:
Hey @modkaffes.. It looks as though the `config` settings are not being referenced here. See error below.

Could you make the update to the Pull Request? Thank you sir. :)

```
ReferenceError: config is not defined
    at subscribe.post (/Users/guy/sites/universe.engineering/routes/subscribe/index.js:34:30)
    at Layer.handle [as handle_request] (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/layer.js:95:5)
    at /Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:47:12)
```